webapp/channels/src/components/channel_info_rhs/menu.tsx

```diff
@@ -171,26 +172,31 @@ export default function Menu(props: MenuProps) {
                     onClick={() => actions.showChannelMembers(channel.id)}
                 />
             )}
-            <MenuItem
-                icon={<i className='icon icon-pin-outline'/>}
-                text={formatMessage({
-                    id: 'channel_info_rhs.menu.pinned',
-                    defaultMessage: 'Pinned messages',
-                })}
-                opensSubpanel={true}
-                badge={channelStats?.pinnedpost_count}
-                onClick={() => actions.showPinnedPosts(channel.id)}
-            />
-            <MenuItem
-                icon={<i className='icon icon-file-text-outline'/>}
-                text={formatMessage({
-                    id: 'channel_info_rhs.menu.files',
-                    defaultMessage: 'Files',
-                })}
-                opensSubpanel={true}
-                badge={loadingStats ? <LoadingSpinner/> : fileCount}
-                onClick={() => actions.showChannelFiles(channel.id)}
-            />
+            {/* TODO: Disable these for now in the popout since they won't work without the search RHS */}
```

Comment:
@matthewbirtch You'll notice in the screenshot that there are two menu items missing. Those require the search RHS to work as a pop-out, so for now, I've just turned them off for this Channel Info pop-out.

Reply:
Ok, so that will be brought back as part of the work to bring search view in the popout?